web/src/components/atoms/DownloadButton/index.tsx

```diff
@@ -5,37 +5,69 @@ import Button, { ButtonProps } from "@reearth-cms/components/atoms/Button";
 import Icon from "@reearth-cms/components/atoms/Icon";
 import { useT } from "@reearth-cms/i18n";
 
+export type PreviewType = "GEO" | "GEO3D" | "IMAGE" | "MODEL3D" | "UNKNOWN";
+
+export type AssetFile = {
+  children?: AssetFile[];
+  contentType?: string;
+  name: string;
+  path: string;
+  size: number;
+};
+
+export type Asset = {
+  id: string;
+  createdAt: string;
+  createdBy: string;
+  createdByType: string;
+  file: AssetFile;
+  fileName: string;
+  previewType?: PreviewType;
+  projectId: string;
+  size: number;
+  url: string;
+  threadId: string;
+  comments: Comment[];
+};
+
+export type Comment = {
```

Comment:
Atom is the smallest component that is aware of itself, and should not know about anything outside its scope.
I'm not sure if the `comments` and `Assets` declarations are right to be here, as they represent things outside the `DownloadButton` scope.

Reply:
Yeah, I was just trying to get the asset table UI/UX updated and merged before the demo so was ignoring some things I didn't want to and this is one of them. 
Not sure the DownloadButton is really necessary/an atom. For refactoring I'll want to maybe move this to be a molecule, stop using it in the asset table all together, make it generic, or something else lol

Reply:
@KaWaite , ok, will keep this in mind :+1: .